### ISSUE TITLE
404 Seed (HELP)

### ISSUE THREAD

**Issue by Starblue0** [chec/commercejs-nextjs-demo-store#235]
I downloaded the repository and did Yarn 
I changed  .env.example to .env.

I Filled in my Public and Secret API Keys for the .env file and I Double checked after every retry and I did request new Priv & Pub API keys when retrying for the seeds and refilled the Env file with new API keys after every retry. 

I ran the yarn seed and I got a 404 error & Licence error in package.json, It was unable to find the seed when I ran the command. 

I tried Updating Windows, Node, Yarn, Visual Studio Code and I tried again and got the same error. 

I am on Windows 11 and it is recently updated and the computer has been restarted. 

I deleted the package & yarn lock file and ran yarn again and got the same error. 

I downloaded the Zip File this time and changed the env and ran yarn and no change same result. 

Kept giving a 404 error and a no license error for the package.json, I did double check and it does show the Bsd-3.

So I then tried using the Chec Cli, I logged into my chec account using the Chec-CLI and then I ran the Chec demo-store command and selected this repo for the full store.  

It said it Downloaded and put the file on my drive but when I looked no file was their, nothing showed using the directory url it showed. 

Tried the same command instead of just on the desktop but in a new file and same result Nothing was downloaded into the new file. 

I have used and tried PowerShell, Command Prompt, VS Code's (Gitbash-CLI, PS, CMD, Node) for running the Yarn Seed command and still end up with the same result as mentioned earlier. 

TL;DR: 

So I cannot get seeds read as it shows a 404, Package.json License Error, Chec-Cli not downloading any Demo store into Local Directory. 

updated and restarted everything and no change in getting the seeds read. I double checked the API Keys and they are correct in the Env file, changed and no result. 

Please help Thank you.
